mdflynn/overlook
# Feature/styling

## What is the change?
- Adds background and animations
- Finishes styling and formatting
- Final format for files
- Add tests for dynamic class methods
- Adds chart for display % full

## What does it fix?
No fix this PR

## Is this a fix or a feature?
This is a feature

## Where should the reviewer start?
- `_mixin.css` Lines 19-44
- `guest-test.js` Whole file
- `user-test.js` Whole file
- `manager-test.js` Whole file

## How should this be tested?
Two ways for this PR.
- Run `npm test` in your terminal to see a passing test suite
- Run `npm start` in your terminal, then navigate to localhost:8080 to see site functionality.
